be/src/env/env_s3.cpp

```diff
@@ -77,13 +77,16 @@ StatusOr<std::unique_ptr<RandomAccessFile>> EnvS3::new_random_access_file(const
     std::string object = path.substr(namenode.size(), path.size() - namenode.size());
     Aws::Client::ClientConfiguration config;
     config.scheme = Aws::Http::Scheme::HTTP;
-    if (!config::aws_s3_endpoint.empty()) {
-        config.endpointOverride = config::aws_s3_endpoint;
+    assert(!config::object_storage_endpoint.empty());
```

Comment:
what if `config::object_storage_endpoint` is empty?

Reply:
oss path check added